utils/src/Utils.js

```diff
@@ -98,6 +98,23 @@ class Utils {
         });
     }
 
+    if (config?.exclusion_group) {
```

Comment:
There is no way to put all of this into the utils function and pass in the config as a variable?

Reply:
i think that's how it works, you add the property to the `config` object: `exclusion_group: 'group-1'`

Reply:
Oh I see.  I was thinking the code was added to the YML.  I see it's in the utils.